shardingsphere-agent/shardingsphere-agent-core/src/main/java/org/apache/shardingsphere/agent/core/bytebuddy/transformer/ShardingSphereTransformer.java

```diff
@@ -138,15 +142,29 @@
             return null;
         }
         if (classStaticMethodPoints.size() == 1) {
-            return new ShardingSphereTransformationPoint<>(methodDescription, new ClassStaticMethodAroundInterceptor(pluginLoader.getOrCreateInstance(classStaticMethodPoints.get(0).getAdvice())));
+            if (classStaticMethodPoints.get(0).isOverrideArgs()) {
```

Comment:
maybe 'guard clauses' will be better here ?

Reply:
Here is only simple if-else clause. Maybe we can keep it.